README.md

```diff
@@ -25,40 +25,16 @@ The tool is hopefull self explanatory:
 ## Forking
 You are welcome to fork the project and run your own instance in terms with the LICENSE.
 
-It would be nice if improvments are feed back into the main project with Pull requests but it is not nessesery to do so.
-
-### Adding Special cards
-The creator can only save cards with hexidecimal editkeys and viewkeys
-this allows for non hexidecimal keys to be used for special cards that are
-impossible for anyone to edit even if they extract the editkey from the code.
-Such a key `SPC-404` is used for the special 404 card.
-
-However such keys must be added manuall to the database,
-here is an example for the SPC-404, refrenced in code.
-
-```
-INSERT INTO tsssff_savedcards2 VALUES (
-    'SPC-404',
-    'SPC-404',
-    'card pony female earthpony',
-    'Card Not Found',
-    '404 Error, Pinkie Pie, Not a card',
-    'Oh Look aint this a fun way to show an error message. Yep your card wasnt found. Unless you wanted this.',
-    '"Argh no no no no! How can we play the game if the cards have all gone missing", "I dont think they are under the... wait pinkie thats the ground how did you"    -Impossible Games night',
-    'https://derpicdn.net/img/view/2013/2/17/246336.jpeg',
-    'Art: sirzi, Card:Ripp_');
-```
-
-Alternatlly a card can be created with the editor when the `UPDATE` command used to modify the editkey and viewkey in the database as nessesery.
+It would be nice if improvements are feed back into the main project with Pull requests but it is not necessary to do so.
```

Comment:
Could you correct "feed" to "fed" here while you're at it, and change "Pull" to "pull"?